index.js

```diff
@@ -67,6 +103,27 @@ module.exports = function(app) {
     if (options.PSILCD1) {
       mapToNmea(PSILCD1)
     }
+        if (options.HDT) {
+      mapToNmea(HDT)
+    }
+    if (options.HDM) {
+      mapToNmea(HDM)
+    }
+    if (options.ROT) {
+      mapToNmea(ROT)
+    }
+    if (options.DBK) {
+      mapToNmea(DBK)
+    }
+    if (options.DBS) {
+      mapToNmea(DBS)
+    }
+    if (options.DBT) {
+      mapToNmea(DBT)
+    }
+    if (options.MTW) {
+      mapToNmea(MTW)
+    }
```

Comment:
Not sure if my change is right. I could not figure out how to reference those module vars like 'APB', so I moved them up on the plugin